src/mixed.js

```diff
@@ -112,7 +112,7 @@ const proto = (SchemaType.prototype = {
         } and ${schema._type}`,
       );
     var cloned = this.clone();
-    var next = merge(this.clone(), schema.clone());
+    var next = prependDeep(schema.clone(), this.clone());
```

Comment:
now that this isn't mutating the first value the second `clone` should be unnecessary, can use `cloned` above or just `this`

Reply:
Right. Now uses `this`, plus `cloned` above not needed too. So 2 clones less.